src/upload.js

```diff
@@ -34,7 +35,9 @@ function UploadDirectory(path, opts) {
 
   const formData = new FormData();
   for (const file of walkDirectory(path)) {
-    formData.append(opts.portalDirectoryFileFieldname, fs.createReadStream(file), { filepath: file });
+    let filepath = file;
+    if (opts.removeRootDir) filepath = file.replace(p.normalize(path), "");
```

Comment:
Can you normalize the path before it gets passed to `walkDirectory`? I'm afraid that `walkDirectory` might otherwise return non-normalized paths, in which case this `replace` call wouldn't work.

Reply:
`walkDirectory` uses `path.join()` which returns a normalized path so I think this is unnecessary?
See https://nodejs.org/api/path.html#path_path_join_paths

Reply:
You're right, but I would still feel more comfortable normalizing before we pass `path` into a different function. Maybe it's okay now but we are making assumptions about `walkDirectory` which could change. I like sanitizing inputs as soon as possible in a function but I'll leave it up to you.